### DISCUSSION TITLE
Github actions -Status checks not found

### DISCUSSION THREAD

**Discussion by skumwhirl** [community/community#28087] [Actions]
We are facing an issue with one of our repositories not showing up the "status checks"  due to which we are not able to trigger a Jenkins job which  normally does a Sonar Scan . let me walk you through the git and Jenkins setup on our side. we have configured a webhook(using GitHub Pull Request Builder Plugin) on a repository when we raise a pull request to "release branch" and comment "Scan Pr" on the comment section of pull request it should trigger a Jenkins job which performs the Sonar Scan. we have configured the branch protection rules for "release" branch under which when we enable "Require branches to be up to date before merging" option we are not able to see any default status checks as we see an message as follows "No status checks found". Please help us understand the reason for which we are seeing such message and brief us about the process of enabling the status checks if there is any.

**Comment by epistax1s** [comment]
Could you show the action configuration file?

**Reply by saiakshaypasham** [reply]
hey @BogomolovNikita we don't have any git actions setup .

**Reply by saiakshaypasham** [reply]
@BogomolovNikita gentle reminder

**Reply by saiakshaypasham** [reply]
any update??

**Comment by skumwhirl** [comment]
is there a way I can add a technical person on my team to this discussion? he can explain it better.

**Reply by epistax1s** [reply]
I think the easiest way would be to send him a link to the current discussion.

**Comment by skumwhirl** [comment]
Sai - can you share the screenshot of the configuration file the gitsupport
team is asking for ?
Santhosh K Gottigere
IoT Global Devops Manager | Global Application Architecture | Cloud
Computing CoE at Whirlpool Corporation
T 2694491659 E ***@***.***
W whirlpoolcorp.com <https://www.whirlpoolcorp.com>


On Tue, Aug 2, 2022 at 12:26 PM BogomolovNikita ***@***.***>
wrote:

> Could you show the action configuration file?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/28087#discussioncomment-3309558>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWF67ZFKNI4HCK7BLHSCHG3VXFD4LANCNFSM55LYRNZQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

-- 
*NOTICE: Whirlpool Corporation e-mail is for the designated recipient only 
and may contain proprietary or otherwise confidential information. If you 
have received this e-mail in error, please notify the sender immediately 
and delete the original. Any other use or disclosure of the e-mail by you 
is unauthorized.*

**Comment by skumwhirl** [comment]
PFA.

THanks,
Sai Akshay Pasham

On Tue, Aug 2, 2022 at 11:31 AM Santhosh K Gottigere <
***@***.***> wrote:

> Sai - can you share the screenshot of the configuration file the
> gitsupport team is asking for ?
> Santhosh K Gottigere
> IoT Global Devops Manager | Global Application Architecture | Cloud
> Computing CoE at Whirlpool Corporation
> T 2694491659 E ***@***.***
> W whirlpoolcorp.com <https://www.whirlpoolcorp.com>
>
>
> On Tue, Aug 2, 2022 at 12:26 PM BogomolovNikita ***@***.***>
> wrote:
>
>> Could you show the action configuration file?
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/orgs/community/discussions/28087#discussioncomment-3309558>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AWF67ZFKNI4HCK7BLHSCHG3VXFD4LANCNFSM55LYRNZQ>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***>
>>
>

-- 
*NOTICE: Whirlpool Corporation e-mail is for the designated recipient only 
and may contain proprietary or otherwise confidential information. If you 
have received this e-mail in error, please notify the sender immediately 
and delete the original. Any other use or disclosure of the e-mail by you 
is unauthorized.*

**Comment by skumwhirl** [comment]
[image: image.png]
sorry please ignore previous attachment.

Thanks & Regards
Sai Akshay Pasham

On Tue, Aug 2, 2022 at 11:43 AM Saiakshay Pasham <
***@***.***> wrote:

> PFA.
>
> THanks,
> Sai Akshay Pasham
>
> On Tue, Aug 2, 2022 at 11:31 AM Santhosh K Gottigere <
> ***@***.***> wrote:
>
>> Sai - can you share the screenshot of the configuration file the
>> gitsupport team is asking for ?
>> Santhosh K Gottigere
>> IoT Global Devops Manager | Global Application Architecture | Cloud
>> Computing CoE at Whirlpool Corporation
>> T 2694491659 E ***@***.***
>> W whirlpoolcorp.com <https://www.whirlpoolcorp.com>
>>
>>
>> On Tue, Aug 2, 2022 at 12:26 PM BogomolovNikita ***@***.***>
>> wrote:
>>
>>> Could you show the action configuration file?
>>>
>>> —
>>> Reply to this email directly, view it on GitHub
>>> <https://github.com/orgs/community/discussions/28087#discussioncomment-3309558>,
>>> or unsubscribe
>>> <https://github.com/notifications/unsubscribe-auth/AWF67ZFKNI4HCK7BLHSCHG3VXFD4LANCNFSM55LYRNZQ>
>>> .
>>> You are receiving this because you authored the thread.Message ID:
>>> ***@***.***>
>>>
>>

-- 
*NOTICE: Whirlpool Corporation e-mail is for the designated recipient only 
and may contain proprietary or otherwise confidential information. If you 
have received this e-mail in error, please notify the sender immediately 
and delete the original. Any other use or disclosure of the e-mail by you 
is unauthorized.*
